commands/set.js

```diff
@@ -11,71 +11,70 @@
 exports.run = async (client, message, [action, key, ...value], level) => { // eslint-disable-line no-unused-vars
-  
 
-  // First we need to retrieve current guild settings
+  // Retrieve current guild settings (merged) and overrides only.
   const settings = message.settings;
   const defaults = client.settings.get("default");
   const overrides = client.settings.get(message.guild.id);
   if (!client.settings.has(message.guild.id)) client.settings.set(message.guild.id, {});
   
-  // Secondly, if a user does `-set edit <key> <new value>`, let's change it
+  // Edit an existing key value
   if (action === "edit") {
     // User must specify a key.
     if (!key) return message.reply("Please specify a key to edit");
     // User must specify a key that actually exists!
-    if (!settings[key]) return message.reply("This key does not exist in the settings");
-    // User must specify a value to change.
+    if (!defaults[key]) return message.reply("This key does not exist in the settings");
     const joinedValue = value.join(" ");
+    // User must specify a value to change.
     if (joinedValue.length < 1) return message.reply("Please specify a new value");
     // User must specify a different value than the current one.
     if (joinedValue === settings[key]) return message.reply("This setting already has that value!");
-
+    
     // If the guild does not have any overrides, initialize it.
     if (!client.settings.has(message.guild.id)) client.settings.set(message.guild.id, {});
 
     // Modify the guild overrides directly.
     client.settings.set(message.guild.id, joinedValue, key);
+
+    // Confirm everything is fine!
     message.reply(`${key} successfully edited to ${joinedValue}`);
   } else
   
-  // If a user does `-set del <key>`, let's ask the user if they're sure...
+  // Resets a key to the default value
   if (action === "del" || action === "reset") {
-    if (!key) return message.reply("Please specify a key to delete (reset).");
-    if (!settings[key]) return message.reply("This key does not exist in the settings");
+    if (!key) return message.reply("Please specify a key to reset.");
+    if (!defaults[key]) return message.reply("This key does not exist in the settings");
     if (!overrides[key]) return message.reply("This key does not have an override and is already using defaults.");
-
-    // Throw the 'are you sure?' text at them.
-    const response = await client.awaitReply(message, `Are you sure you want to reset \`${key}\` to the default \`${defaults[key]}\`?`);
+    
+    // Good demonstration of the custom awaitReply method in `./modules/functions.js` !
+    const response = await client.awaitReply(message, `Are you sure you want to reset ${key} to the default value?`);
 
     // If they respond with y or yes, continue.
-    if (["y", "yes"].includes(response)) {
-
-      // We reset the `key` here.
+    if (["y", "yes"].includes(response.toLowerCase())) {
+      // We delete the `key` here.
```

Comment:
The previous lines mention about "resetting" the key, but you change it to "delete", this could cause some confusion with newer developers.